pinot-core/src/main/java/org/apache/pinot/core/operator/docvalsets/TransformBlockValSet.java

```diff
@@ -118,111 +93,181 @@ public int[] getDictionaryIdsSV() {
   public int[] getIntValuesSV() {
     try (InvocationScope scope = Tracing.getTracer().createScope(TransformBlockValSet.class)) {
       recordTransformValues(scope, DataType.INT, true);
-      return _transformFunction.transformToIntValuesSV(_valueBlock);
+      if (!_isNullHandlingEnabled) {
+        return _transformFunction.transformToIntValuesSV(_valueBlock);
+      }
+      Pair<int[], RoaringBitmap> result = _transformFunction.transformToIntValuesSVWithNull(_valueBlock);
```

Comment:
Should we also add `getIntValuesSVWithNull()` etc so that it is consistent with transform function behavior (separate the APIs with or without null handling)